src/test/scala/com/fulcrumgenomics/sv/tools/SvPileupTest.scala

```diff
@@ -1,12 +1,15 @@
 package com.fulcrumgenomics.sv.tools
 
+import com.fulcrumgenomics.FgBioDef.{FilePath, PathPrefix, PathToBam}
```

Comment:
1. Updated tests to test that we tracked the `SamRecord`(s) correctly
2. Added an end-to-end test to check the SAM tags are added to the correct records (not all within a template)